apps/api/v2/src/modules/api-key/api-key.service.ts

```diff
@@ -19,6 +19,11 @@ export class ApiKeyService {
     response.locals.apiKey = key;
   };
 
+  getApiKeyFromRequest(request: Request): string | null {
+    const apiKey = request.get("Authorization")?.replace("Bearer ", "");
+    return apiKey;
+  }
+
   async retrieveApiKey(request: Request, response?: Response) {
```

Comment:
Since we now have `getApiKeyFromRequest`, it sounds similar to the `retrieveApiKey` below. Probably best to rename `retrieveApiKey` -> `retrieveCalApiKey` so that there is no confusion with the newly added `getApiKeyFromRequest`.